packages/api/src/routes/medical/internal-patient.ts

```diff
@@ -755,4 +775,108 @@ router.post(
   })
 );
 
+/** ---------------------------------------------------------------------------
+ * POST /internal/patient/ops/coverage-assessment
+ *
+ * return the coverage
+ * @param req.query.cxId The customer ID.
+ * @param req.params.id The patient ID.
+ * @param req.query.facilityId The facility ID for running the coverage assessment.
+ * @param req.query.dryrun Whether to simply validate or run the assessment.
+ * @return A patient.
+ *
+ */
+router.post(
+  "/ops/coverage-assessment",
+  requestLogger,
+  asyncHandler(async (req: Request, res: Response) => {
+    const cxId = getUUIDFrom("query", req, "cxId").orFail();
+    const facilityId = getFrom("query").orFail("facilityId", req);
+    const dryrun = getFromQueryAsBoolean("dryrun", req);
+    const payload = coverageAssessmentSchema.parse(req.body);
+
+    if (dryrun) return res.status(status.OK);
+
+    const facility = await getFacilityOrFail({ cxId, id: facilityId });
+    const patientCreates: PatientCreateCmd[] = payload.patients.map(patient => {
+      const phone1 = patient.phone1 ? normalizePhoneNumberStrict(patient.phone1) : undefined;
+      const email1 = patient.email1 ? normalizeEmailStrict(patient.email1) : undefined;
+      const phone2 = patient.phone2 ? normalizePhoneNumberStrict(patient.phone2) : undefined;
+      const email2 = patient.email2 ? normalizeEmailStrict(patient.email2) : undefined;
+      const contact1 = phone1 || email1 ? { phone: phone1, email: email1 } : undefined;
+      const contact2 = phone2 || email2 ? { phone: phone2, email: email2 } : undefined;
+      const contact = [contact1, contact2].flatMap(c => c ?? []);
+      const externalId = patient.externalid ? normalizeExternalId(patient.externalid) : undefined;
+      return {
+        cxId,
+        facilityId: facility.id,
+        externalId,
+        firstName: toTitleCase(patient.firstname),
+        lastName: toTitleCase(patient.lastname),
+        dob: normalizeDate(patient.dob),
+        genderAtBirth: normalizeGender(patient.gender),
+        address: [
+          {
+            addressLine1: toTitleCase(patient.addressline1),
+            addressLine2: patient.addressline2 ? toTitleCase(patient.addressline2) : undefined,
+            city: toTitleCase(patient.city),
+            state: normalizeState(patient.state),
+            zip: normalizeZipCode(patient.zip),
+            country: "USA",
+          },
+        ],
+        contact,
+      };
+    });
+
+    createCoverageAssessments({
+      cxId,
+      facilityId,
+      patientCreates,
+    }).catch(processAsyncError("createCoverageAssessments"));
+    return res.sendStatus(status.OK);
+  })
+);
+
+/** ---------------------------------------------------------------------------
+ * GET /internal/paitnet/ops/coverage-assessment
+ *
+ * Returns the cx patients for a given facility used for internal scripts
+ * @param req.query.facilityId - The facility ID.
+ */
+router.get(
+  "/coverage-assessment",
```

Comment:
I don't see where this is called. Can you point it out, please? Or maybe this is a left-over of an intermediate iteration?

Reply:
https://github.com/metriport/metriport-internal/pull/2079/files#r1700992717 I had forgot the parallel update on internal.